### ISSUE TITLE
Offline maps - clicking my location is not listed above has wrong behaviour

### ISSUE THREAD

**Issue by Danman7** [FundacionParaguaya/MentorApp#865]
Reported: `I just used geco survey and had an issue with maps offline - I used the option my location is not listed above - it found me via location in barcelona - even though country field was empty and mandatory I was allowed to go forward- I went to the next screen - then I went back to location and it showed the antifogasta map`

**Comment by zommerberg** [comment]
i cant reproduce that

**Comment by stacylorraineprendeville** [comment]
@aprojas can you try to reproduce  - I found this bug
@zommerberg were you offline?

**Comment by stacylorraineprendeville** [comment]
@zommerberg please review again - Easy to reproduce here....

**Comment by zommerberg** [comment]
yeah, i was offline @stacylorraineprendeville

**Comment by zommerberg** [comment]
When i tried to reproduce the issue, it found me in the location that the survey is based on,and this is wrong. This is another bug that we would have to fix, before fixing the issue.

**Comment by Danman7** [comment]
I'm taking over.

**Comment by Danman7** [comment]
Also reported: `I was online, and in the location screen, I saw Gateshead location (in the UK). It should have shown Paraguay. Also, it only showed my real location after clicking on the green location icon twice.` Same general idea.

**Comment by Danman7** [comment]
I'm putting this in all related issues:

After discussion with Stacy, showing her all the map usecases and current scenarios, we descided that the maps' functionality will remain as it is at the moment. There was an issue with wrong coordinates for one of the survey centers which was addding confusion and was fixed in the backend. I'm removing this issue from any sprints and putting it in the icebox, to bedescided if it's going to be worked on.
